Corgibytes.Freshli.Cli/Functionality/Analysis/AnalysisLocation.cs

```diff
@@ -4,14 +4,14 @@ namespace Corgibytes.Freshli.Cli.Functionality.Analysis;
 
 public class AnalysisLocation : IAnalysisLocation
 {
-    [JsonProperty] private readonly string _cacheDirectory;
-    [JsonProperty] private readonly string _repositoryId;
-    [JsonProperty] public readonly string? CommitId;
+    [JsonProperty] public string CacheDirectory { get; }
```

Comment:
We can remove the [JsonProperty] attribute when the fields are public.

Reply:
oh yeah! Thanks :D